File: PythonLanguage/BasicPrograms/PurchasePrice.py
def main():
    purchase_input = input("Enter the purchase price: ")
    purchase = int(purchase_input)
    if purchase > 100:
        discount = 0.10 * purchase
        final_price = purchase - discount
    else:
        final_price = purchase
    print("The final price is " + str(final_price))

if __name__ == "__main__":
    main()

File: CLanguage/PurchasePrice.c
#include <stdio.h>

    int main() {
    int purchase;
    int discount;
    int final_price;
    printf("Enter purchase price ");
    scanf("%d", &purchase);
    if (purchase > 100){
        discount = 0.10 * purchase;
        final_price = purchase - discount;
    }
    else{
        final_price = purchase;
    }
    printf("%d",final_price);
    return 0;
}